scripts/run_program.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import numpy as np
+
+d = np.array([1,2,3,4,5,6,7,8,9])
```

Comment:
you should call your variables in a more good way, Peter would not approve this naming